doc/Rules.md

```diff
@@ -127,7 +127,7 @@ Setext style H2
 
 Note: the configured heading style can be a specific style to use (atx,
 atx_closed, setext, setext_with_atx, setext_with_atx_closed), or simply require
-that the usage be consistent within the document.
+that the usage is consistent within the document.
```

Comment:
"require that X be Y" is [correct English](https://ell.stackexchange.com/a/46833/107236).

Reply:
Agree, but I think the proposed change may be more understandable.

Reply:
I'm no expert, not a native English speaker, so take this with a grain of salt, but I think "require that X is Y" is actually incorrect. If an alternative phrasing is preferred, I'd suggest "require the usage to be consistent". WDYT?

Reply:
I just checked on an AI grammar checker and it seems to say that is consistent is better than be consistent.

Reply:
I hardly would trust the output of an AI grammar checker over the assessment of native human speakers. @DavidAnson any thoughts about my suggestion above?